ember/app/components/aircraft-model-select.js

```diff
@@ -10,9 +12,7 @@ export default Ember.Component.extend({
     return [{ id: null }].concat(this.get('models'));
   }),
 
-  model: Ember.computed('modelsWithNull.[]', 'modelId', function() {
-    return this.get('modelsWithNull').findBy('id', this.get('modelId') || null);
-  }),
+  model: findBy('modelsWithNull', raw('id'), 'modelId'),
```

Comment:
Should this be `findBy('modelsWithNull', raw('id'), or('modelId', null))` to match the logic?